cpp/src/json/json_path.cu

```diff
@@ -521,6 +521,14 @@ struct path_operator {
   int index{-1};                          // index for subscript operator
 };
 
+/**
+ * @brief Enum to specify whether parsing values enclosed within brackets, like `['book']`
+ */
+enum class bracket_state : bool {
+  INSIDE,  ///< Parsing inside brackets
+  OUTSIDE  ///< Parsing outside brackets
+};
```

Comment:
I don't think we need such trivial comment.
```suggestion
enum class bracket_state : bool { INSIDE, OUTSIDE };
```

Reply:
We have comments like this for several other enums. I think it's fine to leave alone.

https://github.com/rapidsai/cudf/blob/efc4edfa9dcb30d63379ad23bef23ca330d5bcdf/cpp/include/cudf/types.hpp#L124-L126
https://github.com/rapidsai/cudf/blob/efc4edfa9dcb30d63379ad23bef23ca330d5bcdf/cpp/include/cudf/types.hpp#L116-L118
https://github.com/rapidsai/cudf/blob/efc4edfa9dcb30d63379ad23bef23ca330d5bcdf/cpp/include/cudf/strings/side_type.hpp#L29-L31
https://github.com/rapidsai/cudf/blob/efc4edfa9dcb30d63379ad23bef23ca330d5bcdf/cpp/src/io/json/json_quote_normalization.cu#L43-L45